src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -741,6 +750,7 @@ def algorithm(self, parameters=None):
             # only runs when reqmgr is used (not Tier0)
             self.removeAbortedForceCompletedWorkflowFromCache()
             agentConfig = self.reqAuxDB.getWMAgentConfig(self.config.Agent.hostName)
+            self.enableAllSites = agentConfig.get("SpeedDrainConfig")['EnableAllSites']['Enabled']
```

Comment:
Erik, please update this line to
```
if agentConfig.get("SpeedDrainConfig"):
    self.enableAllSites = agentConfig["SpeedDrainConfig"]['EnableAllSites']['Enabled']
```
to avoid issues when couch is not available.

Reply:
Just a thought - feel free to commit one-liners directly to Erik's branch.  Given the timezone differences, it often makes things simpler to iterate...

Reply:
One line change is enough to break someone else's commit :) However I agree with that if we need to get something in asap